### ISSUE TITLE
Blog Post QA

### ISSUE THREAD

**Issue by vickibrown** [thinkshout/thinkshout#821]
These updates apply to blog post pages https://thinkshout.com/blog/2020/03/critical-digital-focus-covid/

**Top Hero H1 area**
- [x] Change tag color to hex #FFFFFF
- [x] Add underline to tag as hover/focus state
- [x] Change Author hover focus to underline rather than color change
- [x] add 50% more padding beneath hero image

**Body text**
- [x] Change text link color to hex ##BF450D and add underline
- [x] Add hover/focus state to text links to hex #242424

**Below body**
- [x] Next and previous hover/focus state color should be hex #000000
- [x] change tag hover/focus state color to turn hex #000000
_retain underlines on both hover/focus states here_

**Comment by komejo** [comment]
Ready for review on https://a11y-2020--thinkshout.netlify.app/blog/2020/03/critical-digital-focus-covid/

**Comment by vickibrown** [comment]
@komejo looks good!

**Comment by JulesKhong** [comment]
- [ ] The job title text needs to be updated for color contrast and should probably match whatever it's updated to on the Blog Listing page. @vicki What would you recommend?
<img width="131" alt="Screen Shot 2020-05-28 at 3 40 50 PM" src="https://user-images.githubusercontent.com/19519395/83201092-a1cba480-a0f9-11ea-80cf-15b596adc00f.png">

**Comment by vickibrown** [comment]
@JulesKhong lets change those to hex #3E3E3E everywhere. So we'll need to apply this to ticket #820 and #822 as well.

**Comment by komejo** [comment]
Ready for final QA on https://a11y-2020--thinkshout.netlify.app/blog/2020/03/critical-digital-focus-covid/

**Comment by JulesKhong** [comment]
Approved from an a11y QA perspective. Thanks, Joe! 💯

**Comment by vickibrown** [comment]
@komejo looks good!
